### PR TITLE
Responsive tests for CR, MF, and DIEL account signup

### PR DESCRIPTION
**Pull request by abuzar-deriv**
- https://app.clickup.com/t/20696747/QATEST-5533
- https://app.clickup.com/t/20696747/QATEST-5554
- https://app.clickup.com/t/20696747/QATEST-5569
- https://app.clickup.com/t/20696747/QATEST-5827
- https://app.clickup.com/t/20696747/QATEST-6211
- https://app.clickup.com/t/20696747/QATEST-24427

### CODE REVIEW

File: cypress/support/commands/tradersHub.js [hunk 1]
```diff
@@ -300,7 +368,7 @@ Cypress.Commands.add(
           )
         })
       })
-      cy.c_visitResponsive(Cypress.env('verificationUrl'), size)
+      // cy.c_visitResponsive(Cypress.env('verificationUrl'), size)
```
Comment (markodonnell-deriv) [inline cypress/support/commands/tradersHub.js:371]: Please remove comment

Comment (abuzar-deriv) [reply cypress/support/commands/tradersHub.js]: Done.

**Review by markodonnell-deriv — APPROVED** [review]
Look good.

File: cypress/support/commands/tradersHub.js [hunk 2]
```diff
@@ -115,57 +123,88 @@ Cypress.Commands.add(
...
       cy.log('Not IDV or Onfido')
       cy.get('[type="radio"]').first().click({ force: true })
     }
     cy.findByTestId('first_name').type(firstName)
     cy.findByTestId('last_name').type('automation acc')
-    cy.findByTestId('date_of_birth').click()
-    cy.findByText('2006').click()
-    cy.findByText('Feb').click()
-    cy.findByText('9', { exact: true }).click()
+    if (isMobile) cy.findByTestId('date_of_birth').type('2006-02-09')
```
[the remark this answers is not in the document]
Comment (abuzar-deriv) [reply cypress/support/commands/tradersHub.js:150]: Sure @markodonnell-deriv, I can do it once the tests are stable.

File: cypress/e2e/wip/Tradershub/accountSignupMF.cy.js [hunk 3]
```diff
@@ -1,36 +1,50 @@
...
+      cy.c_demoAccountSignup(country, signUpEmail, size)
+      cy.c_generateRandomName().then((firstName) => {
+        cy.c_personalDetails(
+          firstName,
+          'MF',
+          country,
+          nationalIDNum,
+          taxIDNum,
+          currency,
+          { isMobile: size == 'small' ? true : false }
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Tradershub/accountSignupMF.cy.js:25]: why not specify this at the start of the test the way you ar doing it in other spec files.
for all occurrences in this file

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]
One minor change requested

File: cypress/e2e/wip/Tradershub/accountSignupMF.cy.js (hunk 3 above)
Comment (abuzar-deriv) [reply cypress/e2e/wip/Tradershub/accountSignupMF.cy.js]: Had overlooked it. Updated.

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Pull request merged**